### ISSUE TITLE
Kialiv 1.85 encounters a memory leak.

### ISSUE THREAD

**Issue by Chenrujie-85** [kiali/kiali#7968]
### Describe the bug

I deployed kiali on k8s, and after a few days I had a problem with memory usage rising all the time. I'm wondering if the initial configuration has too little memory resources.
By the way, is there any good way to get heap files or other files of kiali's memory usage that can be analyzed with pprof or other tools?
### What are the steps to reproduce this bug?
This is a memory usage graph
![image](https://github.com/user-attachments/assets/c09df2e3-1c33-4190-bed7-7a9e5a6d9333)

### Environment
*Learn about how to determine versions [here](https://kiali.io/docs/faq/general/#how-do-i-determine-what-version-i-am-running).*

Kiali version:1.85
Istio version:1.17.5
Kubernetes impl:
Kubernetes version:1.25.3
Other notable environmental factors:
CPU:1Gi
memory：1Gi

**Comment by jmazzitelli** [comment]
> is there any good way to get heap files or other files of kiali's memory usage that can be analyzed with pprof

Yes, see https://kiali.io/docs/configuration/kialis.kiali.io/#.spec.server.profiler.enabled

**Comment by jshaughn** [comment]
This may be a duplicate of #7800 .

**Comment by jshaughn** [comment]
Thanks for the report, closing as duplicate of #7800, which is being investigated.
